package.json

```diff
@@ -36,6 +37,7 @@
     "amm:kovan:full:migration": "npm run compile && npm run hardhat:kovan -- amm:mainnet --skip-registry",
     "aave:docker:full:migration:add-registry": "npm run compile && npm run hardhat:docker -- aave:mainnet",
     "aave:kovan:full:migration:add-registry": "npm run compile && npm run hardhat:kovan -- aave:mainnet",
+    "aave:goerli:full:migration:add-registry": "npm run compile && npm run hardhat:goerli -- aave:testnet",
```

Comment:
Do we wanna use `aave:testnet` instead of `aave:mainnet`?

Reply:
Yes, due at the moment of the deployment of Aave V2 at the Goerli testnet there was no Chainlink oracles, a main dependency to run the aave:mainnet scripts.

The aave:testnet scripts creates a oracle mock system to impersonate Chainlink oracles.

Reply:
Makes sense, thanks!